2015/music-and-measure-theory/german/sentence_translations.json

```diff
@@ -677,7 +677,7 @@
  {
   "input": "You could imagine that for her, harmonious numbers are precisely those 1% covered by the intervals, provided that her tolerance for error goes down exponentially for more complicated rationals.",
   "model": "nmt",
-  "translatedText": "Man könnte sich vorstellen, dass harmonische Zahlen für sie genau die 1 % sind, die von den Intervallen abgedeckt werden, vorausgesetzt, dass ihre Fehlertoleranz bei komplizierteren rationalen Zahlen exponentiell abnimmt.",
+  "translatedText": "Man könnte sich vorstellen, dass harmonische Zahlen für ihn genau die 1% sind, die von den Intervallen abgedeckt werden, vorausgesetzt, dass ihre Fehlertoleranz bei komplizierteren rationalen Zahlen exponentiell abnimmt.",
```

Comment:
@3b1b you probably deliberately choose to use female pronouns. Switching to German generic masculine is generally ok here but I would like to inquiry upon the original intent since english provides a similar construct but was not chosen. 
Keep explicit female pronouns or switch to generic masculine?

Reply:
If there's no gender-neutral term, I'd err on the side of matching the original. But if that sounds strange or unnatural, or calls excessive attention to itself, then go for what sounds natural. Not speaking German, I find it hard to judge the implications.